gabrielaEnache/src/main/java/collections/store/Order.java

```diff
@@ -12,7 +12,7 @@
     private List<Item> items;
 
     public Order(User user, List<Item> items) {
```

Comment:
as far as I know, You should continue to work on the collection task by adding the OOP task, which comes as an improvement. Let me know if I am wrong?

Reply:
Fixed